app/styles/layout/_status.scss

```diff
@@ -28,6 +28,14 @@ ul.status-list {
       margin: -0.15em 0;
     }
   }
+
+}
+@media only screen and (max-width:400px) {
```

Comment:
Since the styles are "mobile-first", these can be added as the base styles, then a media query added for larger breakpoints.